src/docs/sdk/metrics.mdx

```diff
@@ -63,11 +63,15 @@ class Aggregator:
 
 Normalization shall be done for `statsd` envelope items with unicode support if possible.
 
-* **Namespaces and Units:** regex `[^\w]+` with no replacement character.
-* **Metric Keys/Names:** regex `[^\w\-.]+` with replacement character `_`.
-* **Tag Keys:** regex `[^\w\-.\/]+` with no replacement character.
+* **Namespaces and Units:** regex `[^a-zA-Z0-9_]+` with no replacement character.
+* **Metric Keys/Names:** regex `[^a-zA-Z0-9_\-.]+` with replacement character `_`.
+* **Tag Keys:** regex `[^a-zA-Z0-9_\-.\/]+` with no replacement character.
 * **Tag Values:** See below
 
+**Note:** We can't use `\w` instead of `a-zA-Z0-9` because on some Regex engines `\w` includes
+chars such as `ä`, `ö` or `é`.
+
+
```

Comment:
Let's remove this; it adds more confusion than good.

Reply:
I intended to prevent someone from making the same mistake again. I changed it to an internal note only as a code comment so that it won't be visible in the docs, but only when you edit the .mdx file. Do you still think it is confusing, @cleptric? If yes, I can ditch it.

Reply:
It's fine, thanks!